src/helpers/github.ts

```diff
@@ -99,3 +99,66 @@ export async function isCiGreen({ octokit, logger, env }: Context, sha: string,
     return false;
   }
 }
+
+function parseTarget({ payload }: Context, target: string) {
+  const owner = payload.repository.owner || "";
```

Comment:
```suggestion
  const owner = payload.repository.owner.login || "";
```
`owner` is an object and I'm guessing this is the cause for @Keyrxng problem

Reply:
this seemed to be the issue yeah, I had assumed it was an error on my part. I wanted to manually QA before digging into the code to be sure it wasn't user error 😂